subiquity/common/filesystem/boot.py

```diff
@@ -63,6 +63,9 @@ class MakeBootDevicePlan(abc.ABC):
     this a boot device" in sync.
     """
 
+    def iterate_steps(self):
```

Comment:
Seems like it would be a little cleaner to have a "new_partition_count()" method on MakeBootDevicePlan?